packages/aws-cdk-lib/core/README.md

```diff
@@ -1650,4 +1650,70 @@ warning by the `id`.
 Annotations.of(this).acknowledgeWarning('IAM:Group:MaxPoliciesExceeded', 'Account has quota increased to 20');
 ```
 
+## RemovalPolicys
```

Comment:
Need to make the `RemovalPolicies` change in the README as well :)

Reply:
sorry. I fixed. https://github.com/aws/aws-cdk/pull/32283/commits/af4e0ff9150b4872ea569cd60bd1432d0a851c26